packages/cloud-cognitive/src/components/CreateFullPage/CreateFullPage.stories.js

```diff
@@ -0,0 +1,242 @@
+/**
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+
+import { pkg } from '../../settings';
+import { getStorybookPrefix } from '../../../config';
+
+import { CreateFullPage } from '.';
+import { CreateFullPageSection } from './CreateFullPageSection';
+import { CreateFullPageStep } from './CreateFullPageStep';
+import mdx from './CreateFullPage.mdx';
+
+import styles from './_storybook-styles.scss';
+const storyClass = 'create-full-page-stories';
+
+import { TextInput, NumberInput } from 'carbon-components-react';
+
+const storybookPrefix = getStorybookPrefix(pkg, CreateFullPage.displayName);
+
+export default {
+  title: `${storybookPrefix}/${CreateFullPage.displayName}`,
+  component: CreateFullPage,
+  parameters: {
+    styles,
+    layout: 'fullscreen',
+    docs: { page: mdx },
+    controls: { sort: 'requiredFirst' },
+  },
+  decorators: [
+    (story) => <div className={`${storyClass}__viewport`}>{story()}</div>,
+  ],
+};
+
+const defaultFullPageProps = {
+  nextButtonText: 'Next',
+  backButtonText: 'Back',
+  cancelButtonText: 'Cancel',
+  submitButtonText: 'Submit',
+  modalTitle: 'Are you sure you want to cancel?',
+  modalDescription:
+    "If you cancel, the information you have entered won't be saved.",
+  modalDangerButtonText: 'Cancel partition',
+  modalSecondaryButtonText: 'Return to form',
+  onRequestSubmit: () => {},
```

Comment:
Let's use `action('Submit handler called')` so we know if this is actually getting called at the right time.